thesis-fe/src/components/container/CalendarDashboard.js

```diff
@@ -1,24 +1,188 @@
 import React, { Component } from 'react';
 import styled from 'react-emotion'
-import plus from '../../assets/svg/plus.svg';
+import back from '../../assets/svg/back.svg';
+import { DateRange } from 'react-date-range';
+import { List } from '../container/List';
+
+const Container = styled('div')`
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: #ff7e5f;  /* fallback for old browsers */
+  background: -webkit-linear-gradient(to right, #feb47b, #ff7e5f);  /* Chrome 10-25, Safari 5.1-6 */
```

Comment:
I think we should save the gradient in style constants to make sure we are using the same one